utils/rpms/daos.spec

```diff
@@ -356,6 +357,9 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_libdir}/*.a
 
 %changelog
+* Wed Jun 3 2020 Vishwanath Venkatesan <vishwanath.venkatesan@intel.com> - 1.1.0-20
```

Comment:
you need to change the release at the top to 20 too

Reply:
Done